core/sup_tree_core/executor_pool/executor_pool.ex

```diff
@@ -7,89 +7,118 @@ defmodule AntikytheraCore.ExecutorPool do
   alias Antikythera.ExecutorPool.Id, as: EPoolId
   alias AntikytheraCore.ExecutorPool.Setting, as: EPoolSetting
   alias AntikytheraCore.ExecutorPool.RegisteredName, as: RegName
-  alias AntikytheraCore.ExecutorPool.{ActionRunner, AsyncJobRunner, TimedJobStarter, WebsocketConnectionsCounter, UsageReporter, MemcacheWriter}
+
+  alias AntikytheraCore.ExecutorPool.{
+    ActionRunner,
+    AsyncJobRunner,
+    TimedJobStarter,
+    WebsocketConnectionsCounter,
+    UsageReporter,
+    MemcacheWriter
+  }
```

Comment:
I guess that the reason why this becomes multi lines is the original line is too long.
I don't think both of too long alias and `{}` multiline alias is good.
Then, if we see this type of alias, we should use single aliases.

Reply:
>I guess that the reason why this becomes multi lines is the original line is too long.

I also think so.
According to [here](https://github.com/elixir-lang/elixir/blob/4fea9cc942f632d5a1692eb85774d097546bc4e5/lib/elixir/lib/code.ex#L426) , `{}` seems to be the only place in which `formatter` can introduce a new line safely.